eslint/react/.eslintrc

```diff
@@ -18,13 +18,64 @@
 	},
 	"rules": {
 		"class-methods-use-this": "off",
+		"jsx-quotes": "error",
+		"react/boolean-prop-naming": "error",
+		"react/button-has-type": "error",
+		"react/default-props-match-prop-types": "error",
+		"react/jsx-boolean-value": ["error", "never"],
+		"react/jsx-child-element-spacing": "error",
+		"react/jsx-closing-bracket-location": "error",
+		"react/jsx-closing-tag-location": "error",
+		"react/jsx-curly-brace-presence": "error",
+		"react/jsx-curly-spacing": ["error",  {
+			"when": "always",
+			"children": true
+		}],
+		"react/jsx-equals-spacing": ["error", "never"],
+		"react/jsx-filename-extension": "error",
+		"react/jsx-first-prop-new-line": "error",
+		"react/jsx-fragments": ["error", "element"],
+		"react/jsx-handler-names": "error",
+		"react/jsx-no-bind": "error",
+		"react/jsx-props-no-multi-spaces": "error",
+		"react/jsx-one-expression-per-line": "error",
```

Comment:
pfff, i don't know this one

Reply:
what about allowing single child?

Reply:
I agree, sometimes is not that bad to have multiple items but I think the draw back of forcing to only one is much better than leaving it open and having shit code like:
``` javascript
<App>
  <Hello /> { this.world() }
</App>
```